src/test/resources/FindUser.feature

```diff
@@ -0,0 +1,31 @@
+Feature:
+  As a user
+  I want to be find another user
+
+  Background:
+    Given database is empty
+    And user with id 1234 exists and has username "kangaroo", password "pig-system" and refreshToken "token1"
+    And user with id 1235 exists and has username "penguin", password "i-love-capitalism" and refreshToken "token2"
+    
+  Scenario: Successful find another user
+    When user with id "1234" and token "token1" searches user with search-value "penguin"
```

Comment:
The Endpoint wont be requested with refreshTokens. use the given step `accessToken with value {string} exists for user {long}`. And then request the endpoints with these tokens. You can create a step if you want like `user exists with id id username kevin`, Instead of using the step `user with id {long} exists and has username {string}, password {string} and refreshToken {string}`